### PR TITLE
Update priors spatial main

### PR DESCRIPTION
**Pull request by kaitejohnson**
Merging the changes of infection feedback priors in main into spatial main.

This now reflects the latest stage of the model, which will be featured in a second release (`v0.1.1`) by end of day today. @cbernalz  @arranhamlet

### CODE REVIEW

File: NEWS.md
Comment (dylanhmorris) [inline NEWS.md:1]: Looks like merge conficts made it in here @kaitejohnson

Comment (kaitejohnson) [reply NEWS.md:1]: whoops just fixed

**Review by cbernalz — APPROVED** [review]
Looks good to me!

**Review by dylanhmorris — APPROVED** [review]
LGTM

**Pull request merged**